packages/compass-import-export/src/import/import-csv.spec.ts

```diff
@@ -778,43 +738,35 @@ describe('importCSV', function () {
       errorCallback,
     });
 
-    expect(result.dbStats.insertedCount).to.equal(0);
+    expect(result.docsWritten).to.equal(0);
 
     expect(progressCallback.callCount).to.equal(3);
-    expect(errorCallback.callCount).to.equal(1); // once for the batch
+    expect(errorCallback.callCount).to.equal(3);
```

Comment:
There was this inconsistency where it was one error callback per batch of errors in some cases. It is always one per error now.